src/prefect/telemetry/services.py

```diff
@@ -0,0 +1,67 @@
+from abc import abstractmethod
+from typing import Union
+
+from opentelemetry.exporter.otlp.proto.http._log_exporter import OTLPLogExporter
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk._logs import LogData
+from opentelemetry.sdk._logs.export import LogExporter
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter
+
+from prefect._internal.concurrency.services import BatchedQueueService
+
+
+class BaseQueueingExporter(BatchedQueueService):
+    _max_batch_size = 512
+    _min_interval = 5.0
```

Comment:
Will 5 seconds make things a little choppy for things like watching a flow run in real time?  Any harm in lowering this a bit?

Reply:
We can lower it a bit, 2s ?